addOns/directorylistv1/src/main/javahelp/help/contents/directorylistv1.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+Directory List v1.0
+</TITLE>
+</HEAD>
+<BODY BGCOLOR="#ffffff">
+<H1>Directory List v1.0</H1>
+Provides a file with directory names to be used with "Forced Browse" add-on.
```

Comment:
or fuzzer?

Reply:
Right, this is the description of the add-on, will tweak that too.

Reply:
Tweaked.